### ISSUE TITLE
zsh autocompletion broken

### ISSUE THREAD

**Issue by phw** [flatpak/flatpak#1932]
## Linux distribution and version
Arch Linux (up to date)

## Flatpak version
0.99.3

## Description of the problem
Using the zsh command auto completion for Flatpak fails with the following error:

```
_flatpak:read:1: option valid only in functions called via compctl
_flatpak:read:3: option valid only in functions called via compctl
```

## Steps to reproduce
1. Type `flatpak ` in a zsh prompt
2. Hit the `Tab` key
3. The above error message is shown

**Comment by barthalion** [comment]
Does it also happen on zsh 5.4.2?

**Comment by memnoth** [comment]
It works well on zsh 5.5.1, Antergos which is based on arch.

**Comment by alexlarsson** [comment]
I have no idea about zsh completion. Anyone wanna help?

**Comment by phw** [comment]
I only tested with the version from Arch which is 5.5.1.

@barthalion What version of flatpak? Can you give the contents of `/usr/share/zsh/site-functions/_flatpak`?

Mine is identical to https://github.com/flatpak/flatpak/blob/95ca8a8232bf6d885fe5edf4e51c945e111cae91/completion/_flatpak

**Comment by phw** [comment]
I found the issue, it was my fault, sorry. I still had compctl auto completion configured in my ` .zshrc` as described in https://github.com/flatpak/flatpak/issues/103#issuecomment-271386122 (and obviously forgot about doing this).

Now when Flatpak was released with built-in zsh completion support this got into conflict. But all I noticed was completion working before and being broken after update :)

I took the opportunity to clean up my `.zshrc`
